### DISCUSSION TITLE
why model_config is logged as parameters?

### DISCUSSION THREAD

**Discussion by kyuwoo-choi** [mlflow/mlflow#13896] [General]
With `mlflow.pyfunc.log_model(..., model_config: { "conf1": "value1" }, ...)`, the values in the model_config are logged as parameters. As I understand it, model_config and parameters represent different concepts and use cases. So, why is model_config logged as parameters? Additionally, how can we prevent model_config from being logged as parameters?

**Comment by kevkle** [comment]
As i see it in the moment when a model is registered the model just saves the model that has been trained and for this particular model when you register the model the source run from where it was loaded is linked to the model, i.e. you can always see the parameters in the run.

The reason (I guess) is when you get better version of the models with different params and you want to keep track of the evolution then you would like to see also the parameters which were used in the versions from before and i dont see a way right now how to do this in the model view
